Doc/library/site.rst

```diff
@@ -178,8 +178,9 @@ Module contents
    :func:`getusersitepackages` hasn't been called yet.  Default value is
    :file:`~/.local/lib/python{X.Y}/site-packages` for UNIX and non-framework Mac
    OS X builds, :file:`~/Library/Python/{X.Y}/lib/python/site-packages` for Mac
-   framework builds, and :file:`{%APPDATA%}\\Python\\Python{XY}\\site-packages`
-   on Windows.  This directory is a site directory, which means that
+   framework builds, :file:`{sys.prefix}/lib/python{X.Y}/site-packages` for
+   VxWorks, and :file:`{%APPDATA%}\\Python\\Python{XY}\\site-packages` on
+   Windows. This directory is a site directory, which means that
```

Comment:
Would mind to try to write the different cases as a bullet point for readability?

```
Default value:

* :file:`~/.local/lib/python{X.Y}/site-packages` on UNIX and non-framework macOS builds.
* :file:`~/Library/Python/{X.Y}/lib/python/site-packages` on macOS framework builds.
* :file:`{%APPDATA%}\\Python\\Python{XY}\\site-packages` on Windows.
* :file:`{sys.prefix}/lib/python{X.Y}/site-packages` on VxWorks.

This directory is a site directory,  (...)
```

Same request for USER_BASE below.

Note: if you update the doc, please replace "Mac OS X" with "macOS" (as I did).

Reply:
Done.